rpm/c-ares.spec

```diff
@@ -0,0 +1,70 @@
+Summary: A library that performs asynchronous DNS operations
+Name: c-ares
+Version: 1.16.1
+Release: 1
+License: MIT
+URL: http://c-ares.haxx.se/
+Source0: %{name}-%{version}.tar.bz2
+Patch0: 0001-Use-RPM-compiler-options.patch
+BuildRequires: gcc
+BuildRequires: autoconf
+BuildRequires: automake
+BuildRequires: libtool
+
+%description
+c-ares is a C library that performs DNS requests and name resolves
+asynchronously. c-ares is a fork of the library named 'ares', written
+by Greg Hudson at MIT.
+
+%package devel
+Summary: Development files for c-ares
+Requires: %{name} = %{version}-%{release}
+Requires: pkgconfig
+
+%description devel
+This package contains the header files and libraries needed to
+compile applications or shared objects that use c-ares.
+
+%package doc
+Summary: Documentation for c-ares
+BuildArch: noarch
+
+%description doc
+This package contains documentation of the c-ares.
+
+%prep
+%autosetup -p1 -n %{name}-%{version}/%{name}
+
+f=CHANGES ; iconv -f iso-8859-1 -t utf-8 $f -o $f.utf8 ; mv $f.utf8 $f
+
+%build
+autoreconf --force --install
+%configure --enable-shared --disable-static \
+           --disable-dependency-tracking
+%{__make} %{?_smp_mflags}
```

Comment:
`%make_build` could be used.

Reply:
Yeap. will update. Good catch. Btw, baseline is from fedora's spec.